packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -394,3 +352,36 @@ function defaultFormatCreateLabel(input: string) {
     </div>
   );
 }
+
+function CustomIndicatorsContainer(props: IndicatorsContainerProps) {
```

Comment:
```suggestion
type CustomIndicatorsContainerProps = IndicatorsContainerProps & {
  selectProps: SelectPropsWithExtras;
  children: React.ReactNode;
};

function CustomIndicatorsContainer(props: CustomIndicatorsContainerProps) {
```
I think this could use a better typing to avoid type assertions.

Reply:
works!